packages/ui/src/components/JobCard/Details/DetailsContent/DetailsContent.tsx

```diff
@@ -49,9 +50,7 @@ export const DetailsContent = ({ selectedTab, job, actions }: DetailsContentProp
           {t('JOB.SHOW_ERRORS_BTN')} <ArrowDownIcon />
         </Button>
       ) : (
-        <Highlight language="stacktrace" key="stacktrace">
-          {stacktrace.join('\n')}
-        </Highlight>
+        <Highlight language="stacktrace" key="stacktrace" code={stacktrace.join('\n')} />
```

Comment:
Why did you changed the prop name from `children` to `code`?
`JSON` is not code :]

anyway, this is not related to the compact mode, so let's separate it to a diffrent PR.

Reply:
children is very different from a pure string and this is not intented to have inside something like JSX.Element which is what childrens are.

Modifying this file actually has to do with this PR because the ref doesnt work when the card is collapsed and it doesnt show anything, also the current implementation has some sort of delay, now the tabs change the content straight away as seen in the video.

We could change `code` for `text` or something else, you name it! :)

And yeah Im happy creating an atomic PR for this only component before everything else.